tests/test_base36.nim

```diff
@@ -0,0 +1,120 @@
+import unittest2
+import ../stew/base36
+
+func hexToBytes(a: string, output: var openArray[byte]) =
+  doAssert(len(a) == 2 * len(output))
+  var i = 0
+  var k = 0
+  var r = 0
+  if len(a) > 0:
+    while i < len(a):
+      let c = a[i]
+      if i != 0 and i mod 2 == 0:
+        output[k] = r.byte
+        r = 0
+        inc(k)
+      else:
+        r = r shl 4
+      case c
+      of 'a'..'f':
+        r = r or (10 + ord(c) - ord('a'))
+      of 'A'..'F':
+        r = r or (10 + ord(c) - ord('A'))
+      of '0'..'9':
+        r = r or (ord(c) - ord('0'))
+      else:
+        doAssert(false)
+      inc(i)
+    output[k] = r.byte
+
+func fromHex(a: string): seq[byte] =
```

Comment:
maybe reimplements https://github.com/status-im/nim-stew/blob/d7a6868ba84165e7fdde427af9a1fc3f5f5cc151/stew/byteutils.nim#L188-L199
and could just call it instead?